builtin/providers/azurerm/resource_arm_network_interface_card.go

```diff
@@ -252,6 +258,17 @@ func resourceArmNetworkInterfaceRead(d *schema.ResourceData, meta interface{}) e
 		}
 	}
 
+	if iface.IPConfigurations != nil && len(*iface.IPConfigurations) > 0 {
+		var privateIPAddress *string
+		for _, ipConfig := range *iface.IPConfigurations {
+			privateIPAddress = ipConfig.Properties.PrivateIPAddress
```

Comment:
we are ranging over a collection here and adding each private ip address in an IPConfig to the same string variable

Should the schema have `private_ip_address` as a List? Or can we guarantee that only 1 private IP will be in place?